api/api-pastis/pastis-external/src/main/java/fr/gouv/vitamui/pastis/server/service/PastisService.java

```diff
@@ -150,13 +155,19 @@ public PastisService(
         this.puaFromJSON = puaFromJSON;
     }
 
-    public String getArchiveProfile(final ElementProperties json) throws TechnicalException {
+    public String getArchiveProfile(final ElementProperties json, ProfileVersion version) throws TechnicalException {
         // Recover a statically generated BaliseXML by buildBaliseXMLTree
         json.initTree(json);
         BaliseXML.buildBaliseXMLTree(json, 0, null);
         // Add Recip struct to xml balises tree
         BaliseXML.addRecipTags();
         BaliseXML eparentRng = BaliseXML.getBaliseXMLStatic();
+        // Change namespaces to match seda version
+        if (eparentRng instanceof GrammarXML) {
+            GrammarXML grammarXML = (GrammarXML) eparentRng;
```

Comment:
On peut simplifier :
```suggestion
        if (eparentRng instanceof GrammarXML grammarXML) {
```